scripts/monitoring/hawk_client.yaml.example

```
@@ -0,0 +1,26 @@
+---
+host:
+    name: hostname.example.com
+hawk:
+    url: https://172.17.0.164
+    user: admin
+    pass: XXXXXX
+    verbose: False
+    debug: False
+    ssl_verify: False
+    inactive: True
```

Default is `inactive: True` -> the client lib will fail silently